config/cable.yml

```diff
@@ -0,0 +1,10 @@
+development:
+  adapter: async
+
+test:
+  adapter: async
+
+production:
+  adapter: redis
+  url: redis://localhost:6379/1
+  channel_prefix: screaming_dinosaur_production
```

Comment:
It may be worth also considering just not adding ActionCable if we don't have a use for it. That is, one could imagine we might have generated a *new* Rails 5.1 screaming-dinosaur with it excluded (`rails new screaming-dinosaur -C ...`).